extensions/amp-youtube/0.1/amp-youtube.js

```diff
@@ -234,6 +296,73 @@ class AmpYoutube extends AMP.BaseElement {
       });
     });
   }
+
+  // VideoInterface Implementation. See ../src/video-interface.VideoInterface
+
+  /**
+   * @override
+   */
+  supportsPlatform() {
+    return true;
+  }
+
+  /** @override */
+  isInteractive() {
+    // YouTube videos are always interactive. There is no YouTube param that
+    // makes the video non-interactive. Even data-param-control=0 will not
+    // prevent user from pausing or resuming the video.
+    return true;
+  }
+
+  /**
+   * @override
+   */
+  play(unusedIsAutoplay) {
+    this.playerReadyPromise_.then(() => {
+      this.sendCommand_('playVideo');
```

Comment:
Depends on the platform, on mobile browsers will prevent YT from playing the video unless it is muted. Desktop is fine. (Video manager handles muting videos first before calling play for autoplay case)